### PR TITLE
Chore: Docs

### LINKED ISSUE #28: Chore: Add documentation
**Some options**

- [docz](https://github.com/doczjs/docz)
- [mdx-docs](https://github.com/jxnblk/mdx-docs)
- [react-docgen](https://github.com/reactjs/react-docgen)

### PR DESCRIPTION
**Pull request by wesleimp**
Fixed #28

### CODE REVIEW

**Comment by wesleimp** [comment]
Fixed #28

File: docs/use-map.mdx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  return (
+    <>
+      <p>Map has value for 'key': {map.has('key').toString()}</p>
+      <button onClick={toggleExistence}>Toggle 'Key'</button>
+    </>
+  );
+}
+```
+
+## Display
```
Comment (zaguiini) [inline docs/use-map.mdx]: Change this section's name to "Example"

File: doczrc.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  title: "useDataStructures",
```
Comment (zaguiini) [inline doczrc.js]: Single quotes here

File: docs/use-map.mdx (hunk 1 above)
Comment (zaguiini) [reply docs/use-map.mdx]: Please add one more `#` so we don't get this submenu here:

image

File: docs/use-bubble-sort.mdx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  const unsortedArray = [64, 25, 12, 22, 11];
+  const sortedArray = useBubbleSort(unsortedArray);
+  // you can pass a custom predicate as the second argument
+  // to change how the array will be sorted
+  // (e.g. (a, b) => a > b) to sort the list in descending order
+
+  return (
+    <div>
+      <h2>Unsorted array</h2>
+      <p>{ unsortedArray.join(",").toString() }</p>
```
Comment (zaguiini) [inline docs/use-bubble-sort.mdx]: Add a space on the join argument (like so `",  "`) so the list gets prettier

File: docs/use-bubble-sort.mdx [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+      <h2>Sorted array</h2>
+      <p>{ sortedArray.join(",").toString() }</p>
```
Comment (zaguiini) [inline docs/use-bubble-sort.mdx]: Read above

File: docs/use-bubble-sort.mdx [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+<Playground>
+  {() => {
+    const unsortedArray = [64, 25, 12, 22, 11];
+    const sortedArray = useBubbleSort(unsortedArray);
+
+    return (
+      <div>
+        <h2>Unsorted array</h2>
+        <p>{ unsortedArray.join(",").toString()}</p>
```
Comment (zaguiini) [inline docs/use-bubble-sort.mdx]: Read above

File: docs/use-bubble-sort.mdx [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+
+        <h2>Sorted array</h2>
+        <p>{ sortedArray.join(",").toString()}</p>
```
Comment (zaguiini) [inline docs/use-bubble-sort.mdx]: Read above

File: docs/use-bubble-sort.mdx (hunk 3 above)
Comment (wesleimp) [reply docs/use-bubble-sort.mdx]: Done

File: docs/use-bubble-sort.mdx (hunk 5 above)
Comment (wesleimp) [reply docs/use-bubble-sort.mdx]: Done

File: docs/use-bubble-sort.mdx (hunk 6 above)
Comment (wesleimp) [reply docs/use-bubble-sort.mdx]: Done

File: docs/use-map.mdx (hunk 1 above)
Comment (wesleimp) [reply docs/use-map.mdx]: Done

File: docs/use-bubble-sort.mdx (hunk 4 above)
Comment (wesleimp) [reply docs/use-bubble-sort.mdx]: Done

File: doczrc.js (hunk 2 above)
Comment (wesleimp) [reply doczrc.js]: Done

**Pull request merged**